data-core/MessageFacilityConfigurations/MessageFacility.fcl

```diff
@@ -1,15 +1,15 @@
 udp: { 
 	type: 					UDP 
 	threshold: 				DEBUG 
-	filename_delimit: 		"/src"
+	filename_delimit: 		"/tdaq-v3_01_00/"
```

Comment:
In the future a wildcard may be supported by TRACE, such as "/spack/environments/*/"; if not, could use UptdaqOTS.sh to replace with environment variable definition OTS_ENV